modules/npmscripts.config.js

```diff
@@ -540,6 +543,7 @@ module.exports = {
 			'adaptive-media-web',
 			'admin-dxp-theme',
 			'analytics-client-js',
+			'analytics-reports-js-components-web',
```

Comment:
This shouldn't be here, same with the other analytics modules. Everything should ideally have a @liferay/ prefix

Reply:
Done